### ISSUE TITLE
Accessibility: Ensure button has discernible text

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1757]
URL: https://developer.microsoft.com/en-us/graph/graph-explorer

Path:
div[data-focuszone-id="FocusZone81"] > .ms-Pivot-overflowMenuButton.overflowMenuButton-220.ms-Button--hasMenu

Snippet:
<button type="button" class="ms-Button ms-Button--action ms-Button--command ms-Button--hasMenu ms-Pivot-link ms-Pivot-overflowMenuButton overflowMenuButton-220" data-is-focusable="true" aria-expanded="false" aria-haspopup="true" tabindex="-1">

Fix ONE of the following:
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"
